sarc/cli/acquire/storages.py

```diff
@@ -18,22 +20,29 @@ class AcquireStorages:
     cluster_names: list[str] = field(alias=["-c"], default_factory=list)
     dry: bool = False
 
+    tracer = get_tracer("AcquireStorages")
+
     def execute(self) -> int:
         cfg = config()
 
         for cluster_name in self.cluster_names:
-            print(f"Acquiring {cluster_name} storages report...")
+            with self.tracer.start_as_current_span("cluster") as span:
+                span.set_attribute("cluster_name", cluster_name)
+
+                logging.info(f"Acquiring {cluster_name} storages report...")
+
+                cluster = cfg.clusters[cluster_name]
 
-            cluster = cfg.clusters[cluster_name]
+                fetch_diskusage = methods.get(cluster_name, methods["default"])
+                du = fetch_diskusage(cluster)
 
-            fetch_diskusage = methods.get(cluster_name, methods["default"])
-            du = fetch_diskusage(cluster)
+                if not self.dry:
+                    collection = get_diskusage_collection()
+                    collection.add(du)
+                else:
+                    print("Document:")
+                    print(du.json(indent=2))
 
-            if not self.dry:
-                collection = get_diskusage_collection()
-                collection.add(du)
-            else:
-                print("Document:")
-                print(du.json(indent=2))
+                span.set_status(Status(StatusCode.OK))
```

Comment:
Le status OK ne se set pas automatiquement à la fin du `with` si on ne le set pas manuellement?

Reply:
Non, il reste à UNSET

Reply:
Yuk, je me demande pourquoi ils ont fait ça comme ça. Raison de plus pour faire un context manager.

Reply:
done in https://github.com/mila-iqia/SARC/pull/107/commits/c58821f7d85b4c1027caadb8f671cab0ef3b787d